xgi/algorithms/centrality.py

```diff
@@ -307,3 +308,75 @@ def line_vector_centrality(H):
             vc[node].append(c_i[node])
 
     return vc
+
+
+def katz_centrality(H, index=False, cutoff=100):
+    """Returns the Katz-centrality vector of hypergraph H.
+
+    Parameters
+    ----------
+    H : xgi.Hypergraph
+        Hypergraph on which to compute the alpha centralities.
+    index : bool
+        If set to True, will return a dictionary mapping each vector index to a node.
+    cutoff : int
+        Power at which to stop the computation A + alpha * A**2 + alpha**2 * A**3 + ...
+        Default is 100.
+
+    Returns
+    -------
+    c : np.ndarray
+        Vector of the node centralities, sorted by the node indexes.
+    nodedict : dict
+        If index is set to True, nodedict will contain the nodes ids, keyed by their indice in vector c.
+        <c[key]> will be the centrality of node <nodedict[key]>.
+
+    Note
```

Comment:
Check out [this link](https://numpydoc.readthedocs.io/en/latest/format.html#notes) to see how to markup LaTeX within a docstring.

Also, the section title should be `Notes`, note `Note` (sphinx is very fussy about this...)

Reply:
I put variables in between ``, is that ok ?